examples/sandbox/index.ts

```diff
@@ -188,6 +188,16 @@ async function deviceConnected (deviceId) {
   keyring.on(['*', '*', Events.DISCONNECT], async (deviceId) => {
     $keyring.find(`option[value="${deviceId}"]`).remove()
   })
+
+  keyring.on(['Ledger', '0001', 'ledger.none.retry.response'], async ([ deviceId, event ]) => {
```

Comment:
side thought: I think we ought to make this a "first class" event, like `Events.CONNECT`. don't need to do it as part of this PR though.